server_api_swagger.json

```diff
@@ -1755,6 +1755,33 @@
         }
       }
     },
+    "/contentpacks/installed/upload": {
+      "post": {
+        "description": "Upload an pack to marketplace in Demisto",
+        "consumes": [
+          "multipart/form-data"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "summary": "Upload an pack as zip",
+        "operationId": "zippedPackUpload",
```

Comment:
Please use `uploadContentPacks` (same as used in server)

Reply:
done